### ISSUE TITLE
Installation instructions for linux

### ISSUE THREAD

**Issue by Mathadon** [lbl-srg/BuildingsPy#90]
You need tidylib in order to be able to run buildingspy. In fact you need tidylib and its python wrapper. It took me a while to figure this out (again) when reinstalling my system. I propose to add some hints to this in the installation instructions on this page
http://simulationresearch.lbl.gov/modelica/buildingspy/install.html 

Install instructions can be found here:
http://countergram.com/open-source/pytidylib/docs/index.html

To summarise, I had to run:

```
sudo pip install pytidylib
sudo apt-get install tidy
```

to get it to run!

**Comment by mwetter** [comment]
@Mathadon : Thanks. I added it for the next release.

**Comment by thorade** [comment]
related: #51
